app/controllers/org_admin/templates_controller.rb

```diff
@@ -212,40 +251,32 @@ def copy
     def publish
       template = Template.find(params[:id])
       authorize template
-      
-      # Only allow the current version to be updated
-      if template.latest?
-        # Unpublish the older published version if there is one
-        live = Template.live(template.family_id)
-        if !live.nil? and self != live
-          live.published = false
-          live.save!
-        end
+      # Allowing to publish any version here for future compatibility with publishing older versions from the history page
```

Comment:
this might be problematic. Remember, publishing a historic version means that attempting to generate a new version after will fail since family_id and version are already inserted in the templates table

Reply:
ah true. ok. I will put the check in to make sure its the latest